src/common/localization/en-us.ts

```diff
@@ -163,6 +163,12 @@ export const english: IAppStrings = {
             saveSuccess: "Successfully saved ${connection.name}",
             deleteSuccess: "Successfully deleted ${connection.name}",
         },
+        imageCorsWarning: "Warning: When using VoTT in a Web browser, some assets from Bing Image \
+                          Search may not export correctly due to CORS (cross origin resource sharing) restrictions.",
+        blobCorsWarning: "Warning: When using VoTT in a Web browser, assets in Azure Blob Storage will not export  \
```

Comment:
is this about using Blob as source or target connection?  the wording is a bit confusing

Reply:
if you don't setup CORS you won't be able to use azure blob storage as a source or a target.